docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,162 @@
+Read the [GitHub Docs, Dependabot's compatibility scores](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates#about-compatibility-scores) to learn more about Dependabot's badge.
+
+### Built-in to GitHub vs app
+
+Renovate needs app installation or self-hosting.
+
+Dependabot is built-in to GitHub.
+
+### Scheduling
+
+You can set a schedule for Renovate, per dependency, manager, or even a global schedule.
+Read [Renovate scheduling](./key-concepts/scheduling.md) to learn more.
+
+Dependabot has four options that apply at a language level:
+
+- [`schedule.interval`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval)
+- [`schedule.day`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleday)
+- [`schedule.time`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletime)
+- [`schedule.timezone`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletimezone)
+
+### License
+
+Renovate uses the [GNU Affero General Public License](https://github.com/renovatebot/renovate/blob/main/license).
+
+Dependabot uses [The Prosperity Public License 2.0.0](https://github.com/dependabot/dependabot-core/blob/main/LICENSE).
+
+### Programming language of project
+
+Renovate uses TypeScript.
+
+`dependabot-core` uses Ruby.
+
+## Self-hosting a bot
+
+This section explains how to self-host each bot.
+
+### Self-hosting Renovate
+
+You can self-host Renovate on all [officially supported platforms](https://docs.renovatebot.com/#supported-platforms).
+
+If you decide to self-host Renovate, read the items from the [Self-hosting Renovate reading list](./reading-list.md#self-hosting-renovate).
+
+Available [Renovate distributions](./getting-started/running.md#available-distributions):
+
+- npm package (CLI)
+- Docker images
+- GitHub Action
+- GitLab Runner
+- Mend Renovate On-Premises
+- Mend Remediate (commercial offering)
+- Forking Renovate app
+
+### Self-hosting Dependabot
+
+You can self-host Dependabot on other platforms than GitHub.
+
+#### `dependabot-core`
+
+If you want to customize Dependabot, or self-host on another platform, you can use [`dependabot-core`](https://github.com/dependabot/dependabot-core).
+Quote from the `dependabot-core` readme:
+
+> It [`dependabot-core`] currently supports opening Pull Requests against repositories hosted on GitHub, Github Enterprise, Azure DevOps, GitLab, BitBucket, and AWS CodeCommit.
+
+#### `dependabot-script`
+
+The Dependabot team has a community-maintained collection of scripts to start self-hosting Dependabot: [`dependabot-script`](https://github.com/dependabot/dependabot-script).
```

Comment:
```suggestion
The Dependabot team has a community-maintained collection of scripts to start self-hosting Dependabot: [`dependabot-script`](https://github.com/dependabot/dependabot-script) but the repository has included a message warning that the scripts are broken since March 2023.
```